src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -60,11 +76,19 @@ public EditCommand parse(String args) throws ParseException {
         }
         parseTagsForEdit(argMultimap.getAllValues(PREFIX_TAG)).ifPresent(editPersonDescriptor::setTags);
 
-        if (!editPersonDescriptor.isAnyFieldEdited()) {
-            throw new ParseException(EditCommand.MESSAGE_NOT_EDITED);
```

Comment:
See the comment in `EditCommand`, `Line -82 to -85`.

I think this is a helpful exception to throw, since the expected behavior on using `EditCommand` is that something gets edited, which is impossible if no valid edit fields are passed in.